src/backends/backend/triton_backend_manager.cc

```diff
@@ -359,4 +365,31 @@ TritonBackendManager::CreateBackend(
   return Status::Success;
 }
 
+std::unique_ptr<std::unordered_map<std::string, std::vector<std::string>>>
+TritonBackendManager::BackendState()
```

Comment:
Convention would be to return Status from function and add std::unique_ptr<...>* as argument which you then initialize with the map.